packages/app/i18n/en.pot

```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2021-03-03T10:40:06.300Z\n"
-"PO-Revision-Date: 2021-03-03T10:40:06.300Z\n"
```

Comment:
Shouldn't this file have the strings in `FatalErrorBoundary` removed?

Reply:
You're right, fixed!